pandas/tests/indexing/test_loc.py

```diff
@@ -750,6 +750,64 @@ def test_loc_setitem_with_scalar_index(self, indexer, value):
 
         assert is_scalar(result) and result == "Z"
 
+    @pytest.mark.parametrize(
```

Comment:
are these duplicating the frame tests above?

Reply:
As far as I know frame only allows setting the whole column whereas using indexer allows setting the column only for certain rows.
frame:
```
df[["A", "B"]] = 1
```
indexer:
```
df.loc[[0, 1], ["A", "B"]] = 1
```